benchcab/task.py

```diff
@@ -151,11 +149,8 @@ def get_fluxnet_tasks(config: dict, science_config: dict) -> list[Task]:
             sci_conf_key=key,
             sci_config=science_config[key]
         )
-        for id, branch in config["realisations"].items() for site in met_sites for key in science_config
+        for id, branch in config["realisations"].items()
```

Comment:
Should the argument be `realisations` for mimicking the other arguments? Then call it with `realisations=config["realisations"]`. This way we don't have to give the whole config dictionary to this function when we only need one element of it.

I realise this might be better in a separate PR as it's different from met_sites.

Reply:
Yes I agree, we should avoid passing in the whole config dictionary if we can